### ISSUE TITLE
Could not update allennlp

### ISSUE THREAD

**Issue by Arjunsankarlal** [allenai/allennlp#2168]
Already I had 0.6.0 version. When I try to update, I faced this issue.
Collecting allennlp==0.7.0
  Could not find a version that satisfies the requirement allennlp==0.7.0 (from versions: 0.1a6, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.5.1, 0.6.0)
No matching distribution found for allennlp==0.7.0

I tried uninstalling and then installing as well. Same issue exists. pip is upgraded to the latest version. Using python 3.6 version.

OS: MacOS

**Comment by schmmd** [comment]
@Arjunsankarlal what command are you using to update?  By the way, 0.7.0 isn't the latest release.

**Comment by Arjunsankarlal** [comment]
I tried updating with `pip install -U allennlp`, `pip install allennlp --upgrade`. I tried uninstalling completely and installing it again, but the issue persisted. But then I used a different venv which had the latest version installed.

But now I am facing the same issue again in a different system. In a new venv when I executed the command, `pip install allennlp` it installs 0.6.0 version. And when I tried uninstalling and install with `pip install allennlp==0.8.2` I am getting the issue like, 

Could not find a version that satisfies the requirement allennlp==0.8.2 (from versions: 0.1a6, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.5.1, 0.6.0)
No matching distribution found for allennlp==0.8.2
